analysis/pf_dataset.py

```diff
@@ -33,102 +38,117 @@ def get_pf_clinical_events(pharmacy_first_ids):
 # - chronic sinusitis and immunosuppressed individuals for acute sinusitis
 def get_uncomplicated_uti_denominator(index_date, selected_events, pregnancy_codelist):
     urt_code = ["1090711000000102"]
-    count_urt_6m = count_past_events(index_date, selected_events, urt_code, 6)
-    count_urt_12m = count_past_events(index_date, selected_events, urt_code, 12)
+    count_urt_6m = count_past_events(index_date, clinical_events, urt_code, 6)
+    count_urt_12m = count_past_events(index_date, clinical_events, urt_code, 12)
 
     age = patients.age_on(index_date)
```

Comment:
I just realised we should also provide `patients` as an argument to the function